config/application.rb

```diff
@@ -1,6 +1,6 @@
-require_relative 'boot'
+require_relative "boot"
 
-require 'rails/all'
+require "rails/all"
```

Comment:
I would probably have culled the quotation-mark changes but that's not actually important either way

Reply:
Legit, lemme undo them